### PR TITLE
feat(stub): allow for custom secure site stub file

### PR DESCRIPTION
**Pull request by jjpmann**
Fixes #941

### CODE REVIEW

**Comment by drbyte** [comment]
@jjpmann Tip:  if you edit the first post and add `Fixes #` followed by the number of the Issue you opened about this, when the PR is merged it will automatically clean up the repo by closing the associated issue.

File: cli/Valet/Site.php [hunk 1]
```diff
@@ -220,6 +220,20 @@ function getSites($path, $certs)
         });
     }
 
+    /**
+     * Get Stub file for securing site
```
Comment (drbyte) [inline cli/Valet/Site.php:224]: Standard single-line comment syntax in this repo is to end the sentence with a period `.`

**Comment by drbyte** [comment]
When first looking at this I found myself wanting your new function name to include the word "Secure" since it's dealing only with "secure" stubs.

But then it started to feel more like if you pass the name of the stub file wanted, it could serve a broader purpose.

So, if we go with `getSiteStub($filename)` perhaps we can use the same lookup elsewhere.

The caveat to this is that when Valet's default stubs get updated you will have to manually incorporate those changes, because any override stub files you create will be completely orphaned from any of Valet's control ... making support rather nightmarish.

**Comment by drbyte** [comment]
> * Change PORT due to haproxy/varnish

the new proxy command may make this less necessary

**Comment by jjpmann** [comment]
Thanks for the feedback. 

While the proxy command sounds great having to run that each time makes me want to have a global override for the defaults. I assume there are better ways, just something I haven't thought of just yet.

**Comment by drbyte** [comment]
@jjpmann do you mind resolving the merge conflict?

@mattstauffer I think I'm satisfied that we can merge this.

**Comment by mattstauffer** [comment]
@jjpmann Sorry, it looks like we have one more conflict. Would you mind resolving those and then I'll be happy to merge? Sorry for my delay!

**Comment by drbyte** [comment]
Thanks @jjpmann.

cc @mattstauffer

**Comment by mattstauffer** [comment]
Ugh. I'm the worst, and missed this again. Please accept my apologies, @jjpmann -- would you mind updating it one more time?

**Comment by jjpmann** [comment]
Closing in favor of https://github.com/laravel/valet/pull/1238

**Pull request closed without merging**